paasta_tools/long_running_service_tools.py

```diff
@@ -49,6 +50,7 @@
 METRICS_PROVIDER_PISCINA = "piscina"
 METRICS_PROVIDER_ACTIVE_REQUESTS = "active-requests"
 METRICS_PROVIDER_PROMQL = "arbitrary_promql"
+METRICS_PROVIDER_WORKER_LOAD = "worker_load"
```

Comment:
we should probably try to be more consistent with the names here and have this be `worker-load`

(i can make a pr to rename the promql one since nothing is using that yet)